lib/ramble/ramble/spack_runner.py

```diff
@@ -443,6 +460,7 @@ def inventory_hash(self):
         if self.dry_run:
             spack_file = os.path.join(self.env_path, 'spack.yaml')
 
+        return 'test'
```

Comment:
Is this at the right indent, or should it be in the dryrun? Looks like a bit of an overly aggressive early exit?

Reply:
Good catch, this shouldn't be here at all. :)